.gitignore

```diff
@@ -1,6 +1,7 @@
 .clang_complete
 *.o
 *.a
+.*.sw?
```

Comment:
This should be removed.

Reply:
@ascent12 already pointed that out. I'll remove the .gitignore commit when I'm squashing.

Reply:
Oh, sorry about that. Leaving this discussion opened so that we don't forget about it.